### ISSUE TITLE
Lightness Setting

### ISSUE THREAD

**Issue by cww97** [phaselden/FlipIt#41]
I noticed that we have the lightness setting in Fliqlo in case I wanna it when sleeping,
Can Fliplt have this feature ?

**Comment by SmilerRyan** [comment]
Agreed, an option to have the background there and the clock at say 50% brightness would be quite nice.

**Comment by phaselden** [comment]
Sounds like a good option.

**Comment by Sabine5034** [comment]
Hi Phil, is there any chance that you integrate a variable display-dimm-function. I love the app, especially the adjustable digit size. At night with the large digits it is nicely to read the time even from a 3 meters distance. Only the display is too bright. It should be dimmable. Reasonable request?
Thanks and regards,
Ben

**Comment by phaselden** [comment]
Hi, Thanks for the suggestion. I will consider that next time I do a round of updates. 

One thing FYI is that I had considered it previously, but decided not to implement it because my monitor (I didn't check other systems) allowed dimming in the driver or other system settings. Anyway, I will look at it again at some point.

**Comment by Sabine5034** [comment]
Thanks Phil and... Happy New Year from France.
Ben
